sparkle/src/components/Popover.tsx

```diff
@@ -50,13 +50,14 @@ const PopoverContent = React.forwardRef<
 
 interface PopoverProps extends Omit<PopoverContentProps, "content"> {
   trigger: React.ReactNode;
+  triggerIsButton?: boolean;
```

Comment:
`nit`: let's rename it to `popoverAsChild` or `popoverTriggerAsChild`?

Reply:
done!